cocos/renderer/gfx/IndexBuffer.cpp

```diff
@@ -110,11 +111,25 @@ void IndexBuffer::update(uint32_t offset, const void* data, size_t dataByteLengt
     {
 //        glBufferData(GL_ELEMENT_ARRAY_BUFFER, (GLsizeiptr)dataByteLength, data, glUsage);
         glBufferData(GL_ELEMENT_ARRAY_BUFFER, _bytes, (const GLvoid*)data, glUsage);
+        GLenum gl_err = glGetError();
+        if (0 != gl_err){
+            RENDERER_LOGE("GL error 0x%x: %s:%s", gl_err, glEnumName(gl_err), __FUNCTION__);
+            _drawable = false;
```

Comment:
Should initialize it in `init()`.

Reply:
As the object will be destroyed and initialized again.

Reply:
You meant init `gl_err` object? I can't find any example code doing that. How?

Reply:
Sorry, i mean `_drawable`.

Reply:
I see. Gotcha.